index.html

```diff
@@ -503,6 +519,7 @@
                                 priorities.appendChild(box);
                             }
                         }
+                        lowestPriority = priorityName;
```

Comment:
From my quick research yesterday, there is no default priority query. 

I just found something that will work for this and severity